README.md

```diff
@@ -8,6 +8,8 @@ You will also need [mruby-contracts](https://github.com/nervosnetwork/mruby-cont
 
 If you don't want to build mruby-contracts yourself, we have a prebuilt binary at [here](https://github.com/nervosnetwork/binary/raw/master/contracts/mruby/argv_source_entry).
 
+Finally you will need Ruby 2.4+ to run this SDK.
```

Comment:
I'm not so sure about this change since @doitian mentioned that Mac OS X still ships with Ruby 2.3.x

Reply:
@xxuejie First of all, we already have usage of 2.4+ only (`String#unpack1`); second, official support for Ruby 2.3 ends Mar 2019.

I believe it's reasonable to ask developers to stick to 2.4 and above ;-)

(BTW, macOS has tradition to have too old version of Ruby bundled.)

Reply:
Can we change it to `unpack` if `unpack1` is the only incompatible one?
```
unpack('h*')[0]
```

Reply:
Changing to `unpack` looks good to me.

Reply:
Until we find more 2.4 only usage in the future 😄 . That has happened twice.

Reply:
One more thing, the extract `scripts` repo has `unpack1` in multiple places as well.